amarachukwu-agbo/random-number-generator
# #165145102 Set up React and Parcel

#### What does this PR do?
Sets up React and Parcel

#### Description of Task to be completed?
- Install react, react-dom, parcel and babel-preset-react
- Set up scripts to run the client

#### How should this be manually tested?
- Clone the repo and install application dependencies. Run command  `yarn run client:start-dev`.
- Navigate to `http://localhost:1234`
- You should see the app running

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#165145102](https://www.pivotaltracker.com/story/show/165145102)

#### Screenshots (if appropriate)
N/A

#### Questions: 
N/A
